sdk/include/opentelemetry/sdk/metrics/meter.h

```diff
@@ -0,0 +1,109 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+#ifndef ENABLE_METRICS_PREVIEW
+#  include <chrono>
+#  include "opentelemetry/metrics/meter.h"
+#  include "opentelemetry/sdk/instrumentationlibrary/instrumentation_library.h"
+#  include "opentelemetry/sdk/metrics/meter_context.h"
+#  include "opentelemetry/sdk/resource/resource.h"
+#  include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace sdk
+{
+namespace metrics
+{
+class Meter final : public opentelemetry::metrics::Meter
+{
+public:
+  /** Construct a new Meter with the given  pipeline. */
+  explicit Meter(std::shared_ptr<sdk::metrics::MeterContext> context,
+                 std::unique_ptr<opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary>
+                     instrumentation_library =
+                         opentelemetry::sdk::instrumentationlibrary::InstrumentationLibrary::Create(
+                             "")) noexcept;
+
+  nostd::shared_ptr<opentelemetry::metrics::Counter<long>> CreateLongCounter(
+      nostd::string_view name,
+      nostd::string_view description = "",
+      nostd::string_view unit        = "1") noexcept override;
```

Comment:
Consider using null or empty string as the default value.

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#instrument

_If the unit is not provided or the unit is null, the API and SDK MUST make sure that the behavior is the same as an empty unit string._